eng/build-analysis-configuration.json

```diff
@@ -0,0 +1,8 @@
+{
+    "PipelinesToAnalyze":[
+       {
+          "PipelineId": 538524,
```

Comment:
I think you are using a build Id instead of a pipeline Id, I think the pipeline id should be 129, base on https://dev.azure.com/dnceng-public/public/_build?definitionId=129